run--03b--postestimation-disaggregate-estimates.Rmd

```diff
@@ -411,6 +417,11 @@ nowcast_agg_sd    <- agg_tracts_up(nowcast_elig_counts_final, "school")
 if (exists("my_aux_geo")) {
   nowcast_agg_aux <- agg_tracts_up(nowcast_elig_counts_final, "aux_geo_label")  
 }
+
+if (my_state_abbr == "IL") {
```

Comment:
As above -- this shouldn't be in `main`. We should take time to expand the `aux` specification.